src/main/java/uk/gov/justice/hmpps/probationteams/ProbationTeamsApplication.kt

```diff
@@ -9,5 +9,5 @@ import org.springframework.transaction.annotation.EnableTransactionManagement
 open class ProbationTeamsApplication
 
 fun main(args: Array<String>) {
-  runApplication<ProbationTeamsApplication>(*args)
+    runApplication<ProbationTeamsApplication>(*args)
```

Comment:
has this reformatted with 4 spaces instead of 2?

Reply:
just realised that your project settings are for 4 character indent for kotlin :grin: